libraries/core-react/src/Popover/PopoverItem.tsx

```diff
@@ -0,0 +1,199 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import React, { useEffect, useRef, forwardRef, HTMLAttributes } from 'react'
+import styled, { css } from 'styled-components'
+import { Icon } from '../Icon'
+import { Card } from '../Card'
+import { Button } from '../Button'
+import { close } from '@equinor/eds-icons'
+import { spacingsTemplate, typographyTemplate } from '../_common/templates'
+import { useCombinedRefs } from '../_common'
+
+import { popover as tokens, Placement } from './Popover.tokens'
+
+Icon.add({ close })
+
+type WrapperProps = {
+  top: string | number
+  bottom: string | number
+  right: string | number
+  left: string | number
+  transform: string
+}
+
+const StyledPopoverWrapper = styled.div<WrapperProps>`
+  ${({ top, bottom, right, left, transform }) =>
+    css`
+      bottom: ${bottom};
+      top: ${top};
+      right: ${right};
+      left: ${left};
+      transform: ${transform};
+    `}
```

Comment:
Note to self, clean this up with `csstype` in the future. We'll make a seperate issue for this